src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidationResult.java

```diff
@@ -44,6 +44,10 @@ public boolean isValid() {
         return this.isValid;
     }
 
+    public boolean isInvalid() {
```

Comment:
Shall we rename to `isNotValid`

Reply:
Renamed in commit 96d4dc498094daee16277501378ee52fbb484dd1